### PR TITLE
2023 data updates

### PR DESCRIPTION
**Pull request by freyac**
This is an update with the new compliance data released in December 2023. 

There are two loose ends I need to track down by emailing CARB: 
1. There's a facility id (104091) that appears in compliance data but not the contemporary facility data. From my last exchange with CARB, I think this is a result of the MRR data reflecting facility changes more quickly than the compliance data. 
2. There's a project id that appears in the compliance data, but not in the issuance data, which I think is just a typo (CAFR-6339-A).

### CODE REVIEW

**Review by badgley — APPROVED** [review]

File: docs/make-updates.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+## Steps for annual data updates
```
Comment (katamartin) [inline docs/make-updates.md:1]: ✨ 🎉

**Review by katamartin — CHANGES_REQUESTED** [review]
See details in comment below, but I think updating the data used in the tool will require more testing on the frontend (for example, we still need to add 2022 to the [reporting periods filter](https://github.com/carbonplan/compliance-users/blob/f26abf19aad04860f12fdb914dae5454e19f3629/components/use-store.js#L15-L19)).

File: components/use-store.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import create from 'zustand'
 
 const url =
-  'https://raw.githubusercontent.com/carbonplan/compliance-users/main/data/outputs/user_data_v2.0.json'
+  'https://raw.githubusercontent.com/carbonplan/compliance-users/main/data/outputs/user_data_v3.0.json'
```
[the remark this answers is not in the document]
Comment (freyac) [reply components/use-store.js:4]: implementing option 1!

**Review by katamartin — COMMENTED** [review]
Looking much better! Possible to add `2022` to the list of options here?
https://github.com/carbonplan/compliance-users/blob/f26abf19aad04860f12fdb914dae5454e19f3629/components/use-store.js#L15-L19

File: components/use-store.js [hunk 3]
```diff
@@ -1,7 +1,10 @@
 import create from 'zustand'
 
+// const url =
+//   'https://raw.githubusercontent.com/carbonplan/compliance-users/main/data/outputs/user_data_v3.0.json'
```
Comment (katamartin) [inline components/use-store.js:4]: Let's restore this and merge!

**Review by katamartin — APPROVED** [review]
Once we update to use the real URL, I say we merge this 🚢

**Pull request merged**